src/components/plots/FrequencyPlot.jsx

```diff
@@ -23,13 +23,13 @@ const FrequencyPlot = (props) => {
   const [plotSpec, setPlotSpec] = useState({});
 
   useEffect(() => {
-    if (cellSets.loading && !cellSets.error) {
```

Comment:
`cellSets.loading` here was unnecessary, it is being checked inside `loadCellSets` already